test/fail.test.js

```diff
@@ -243,3 +243,130 @@ test.serial("Does not post comments when failComment is set to false", async (t)
+  const gitlab = authenticate(env)
+    .get(`/projects/${encodedRepoId}/issues?state=opened&&search=${encodedFailTitle}`)
+    .reply(200, [
+      {
+        id: 2,
+        iid: 2,
+        project_id: 1,
+        web_url: "https://gitlab.com/test_user/test_repo/issues/2",
+        title: "API should implemented authentication",
+      },
+    ]);
+
+  await fail(pluginConfig, { env, options, branch, errors, logger: t.context.logger });
+
+  t.true(gitlab.isDone());
+});
+
+test.serial("Does not post comments on existing issues when failCommentCondition disables this", async (t) => {
+  const owner = "test_user";
+  const repo = "test_repo";
+  const env = { GITLAB_TOKEN: "gitlab_token" };
+  const pluginConfig = { failCommentCondition: "<% return !issue; %>" };
+  const branch = { name: "main" };
+  const options = { repositoryUrl: `https://gitlab.com/${owner}/${repo}.git` };
+  const errors = [{ message: "An error occured" }];
+  const encodedRepoId = encodeURIComponent(`${owner}/${repo}`);
+  const encodedFailTitle = encodeURIComponent("The automated release is failing 🚨");
+  const gitlab = authenticate(env)
+    .get(`/projects/${encodedRepoId}/issues?state=opened&&search=${encodedFailTitle}`)
+    .reply(200, [
+      {
+        id: 1,
+        iid: 1,
+        project_id: 1,
+        web_url: "https://gitlab.com/test_user%2Ftest_repo/issues/1",
+        title: "The automated release is failing 🚨",
+      },
+      {
+        id: 2,
+        iid: 2,
+        project_id: 1,
+        web_url: "https://gitlab.com/test_user%2Ftest_repo/issues/2",
+        title: "API should implemented authentication",
+      },
+    ]);
+
+  await fail(pluginConfig, { env, options, branch, errors, logger: t.context.logger });
+
+  t.true(gitlab.isDone());
+});
+
+test.serial("Post new issue if none exists yet with disabled comment on existing issues", async (t) => {
+  const owner = "test_user";
+  const repo = "test_repo";
+  const env = { GITLAB_TOKEN: "gitlab_token" };
+  const pluginConfig = { failCommentCondition: "<% return !issue; %>" };
+  const branch = { name: "main" };
+  const options = { repositoryUrl: `https://gitlab.com/${owner}/${repo}.git` };
+  const errors = [{ message: "An error occured" }];
+  const encodedRepoId = encodeURIComponent(`${owner}/${repo}`);
+  const encodedFailTitle = encodeURIComponent("The automated release is failing 🚨");
+  const gitlab = authenticate(env)
+    .get(`/projects/${encodedRepoId}/issues?state=opened&&search=${encodedFailTitle}`)
+    .reply(200, [
+      {
+        id: 2,
+        iid: 2,
+        project_id: 1,
+        web_url: "https://gitlab.com/test_user/test_repo/issues/2",
+        title: "API should implemented authentication",
+      },
+    ])
+    .post(`/projects/${encodedRepoId}/issues`, {
+      id: "test_user%2Ftest_repo",
+      description: `## :rotating_light: The automated release from the \`main\` branch failed. :rotating_light:
```

Comment:
Can we somehow avoid have that lengthy description here? It's not really important for the test case and hurts a bit on the eyes :) No blocker, just if there's an easy way.